### PR TITLE
[UNR-4585] Replay test gym

### PR DESCRIPTION
**Pull request by mironec**
Adds a test gym for testing replays.
Nothing special there, so created a few nothing special characters, since I have often found the need to have really basic characters, rather than the default NFR ones, which spawn cubes and whatnot.

### CODE REVIEW

File: Game/Source/GDKTestGymsEditor.Target.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ public GDKTestGymsEditorTarget(TargetInfo Target) : base(Target)
 		Type = TargetType.Editor;
 		ExtraModuleNames.Add("GDKTestGyms");
 		ExtraModuleNames.Add("GDKTestGymsFunctionalTests");
+
 		DefaultBuildSettings = BuildSettingsVersion.V2;
```
Comment (mironec) [inline Game/Source/GDKTestGymsEditor.Target.cs:14]: Hm, I see I conflicted with changes from master here, so someone had the same idea.
Not sure if it's necessary for all targets?

File: Game/Source/GDKTestGyms/Public/ReplayPlayerController.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```
Comment (mironec) [inline Game/Source/GDKTestGyms/Public/ReplayPlayerController.h:1]: I thought I changed this default at some point...

File: Game/Source/GDKTestGyms/GDKTestGyms.Build.cs [hunk 3]
```diff
@@ -19,6 +19,7 @@ public GDKTestGyms(ReadOnlyTargetRules Target) : base(Target)
 				"GameplayAbilities",
 				"GameplayTasks",
 				"InputCore",
+				"Slate",
```
Comment (mironec) [inline Game/Source/GDKTestGyms/GDKTestGyms.Build.cs:22]: Needed for the input component stuff and binding keys directly.
Maybe I could setup action mappings instead....

File: Game/Source/GDKTestGymsEditor.Target.cs (hunk 1 above)
Comment (m-samiec) [reply Game/Source/GDKTestGymsEditor.Target.cs:14]: I think you're right, I should have set it in all targets

**Review by m-samiec — APPROVED** [review]

**Pull request merged**